overlayfs-tools/makefile

```diff
@@ -1,12 +1,16 @@
 CFLAGS = -Wall -std=c99
 LFLAGS = -lm
+LIBS = -lfts
 CC = gcc
 
 all: overlay
 
 overlay: main.o logic.o sh.o
 	$(CC) $(LFLAGS) main.o logic.o sh.o -o overlay
 
+external: main.o logic.o sh.o
+	$(CC) $(LFLAGS) main.o logic.o sh.o -o overlay $(LIBS)
+
```

Comment:
I'm no make expert either, but to get this to compile I added `-lfts` to LFLAGS and simply put LFLAGS at the end of this line.

`$(CC) main.o logic.o sh.o -o overlay $(LFLAGS)`

I used this SO answer for reference: https://stackoverflow.com/a/13250090

Reply:
Yep that is similar to what i am doing. I am not sure what will happen when i permanlty add the flag. Maybe that's an option, but im not sure if it will still compile on Debian? The only difference between the normal build target (overlay) and the 'external' build target is the addition of -lfts. But by using LIBS, one might add more eternal libs later if needed.